.github/workflows/automatic-doc-checks.yml

```diff
@@ -1,9 +1,11 @@
 name: Main Documentation Checks
 
 on:
-  - push
-  - pull_request
-  - workflow_dispatch
+  push:
+    branches-ignore:    
+      - 'ubuntu-core-devel'
+  pull_request:
+  workflow_dispatch:
```

Comment:
Ignored here to prevent double workflow runs when creating PRs against this branch.

Revert before merging.